### PR TITLE
AT__03.03.03 Verify Clicking "Booking management" opening the page with heading "Booking list"

### PR DESCRIPTION
**Pull request by AlexMokeev**

### CODE REVIEW

**Comment by AlexMokeev** [comment]
https://trello.com/c/bLXXiAVg/236-at030303-verify-clicking-booking-management-opening-the-page-with-heading-booking-list

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js [hunk 1]
```diff
@@ -4,6 +4,10 @@ import LeftMenuPanel from "../../../../pageObjects/LeftMenuPanel";
 
 const leftMenuPanel = new LeftMenuPanel();
 
+import BookingsListPage from "../../../../pageObjects/BookingsListPage";
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:7]: imports need to be grouped moved to line 4

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js [hunk 2]
```diff
@@ -4,6 +4,10 @@ import LeftMenuPanel from "../../../../pageObjects/LeftMenuPanel";
...
+
+const bookingsListPage = new BookingsListPage();
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:9]: const also needs to be moved after const leftMenuPanel

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js [hunk 3]
```diff
@@ -12,6 +16,12 @@ describe('US_03.03 Bookings management link', () => {
         cy.fixture('leftMenuPanel/menuLinks').then(link => {
             this.link = link;
         });
+        cy.fixture('leftMenuPanel/url').then(url => {
+            this.url = url;
+        });
+        cy.fixture('header/headers').then(header => {
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:22]: Header fixture should be used from the bookingsListPage/headers

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js [hunk 4]
```diff
@@ -25,4 +35,14 @@ describe('US_03.03 Bookings management link', () => {
             .getBookingManagementNameLink()
             .should('include.text', this.link.bookingsManagementLink)
     });
+
+    it('AT__03.03.03 Verify Clicking "Booking management" opening the page with heading "Booking list"', function () {
+        leftMenuPanel
+            .clickGetBookingManagementIconLink()
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:41]: do not need to wrap to another line, it is not long

File: cypress/fixtures/leftMenuPanel/url.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+    "bookingsManagementUrl": "https://qatest.site/orders/"
+}
```
Comment (Rom-T) [inline cypress/fixtures/leftMenuPanel/url.json:3]: it is better to transfer the url.json file to the bookingsListPage/url

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js [hunk 6]
```diff
@@ -12,6 +16,12 @@ describe('US_03.03 Bookings management link', () => {
         cy.fixture('leftMenuPanel/menuLinks').then(link => {
             this.link = link;
         });
+        cy.fixture('leftMenuPanel/url').then(url => {
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:19]: bookingsListPage/url

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js (hunk 1 above)
Comment (AlexMokeev) [reply cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:7]: ok

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js (hunk 2 above)
Comment (AlexMokeev) [reply cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:9]: ok

File: cypress/fixtures/leftMenuPanel/url.json (hunk 5 above)
Comment (AlexMokeev) [reply cypress/fixtures/leftMenuPanel/url.json:3]: ok

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js (hunk 6 above)
Comment (AlexMokeev) [reply cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:19]: ok

**Review by Rom-T — APPROVED** [review]

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js [hunk 7]
```diff
@@ -1,8 +1,10 @@
 /// <reference types="Cypress" />
```
Comment (elviraKh1) [inline cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:1]: @Rom-T @AlexMokeev  I have a question about file name  and I  think  US_03.03_BookingsManagementLink.cy.js  more correct

Comment (Rom-T) [reply cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:1]: @elviraKh1 Yes that's right. I didn't notice it

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js [hunk 8]
```diff
@@ -25,4 +33,12 @@ describe('US_03.03 Bookings management link', () => {
             .getBookingManagementNameLink()
             .should('include.text', this.link.bookingsManagementLink)
     });
+
+    it('AT__03.03.03 Verify Clicking "Booking management" opening the page with heading "Booking list"', function () {
+        leftMenuPanel.clickGetBookingManagementIconLink() 
+        cy.url().should('eq', this.url.bookingsManagementUrl)
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:39]: Remove the URL check. Leave only on the header

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js [hunk 9]
```diff
@@ -12,6 +14,12 @@ describe('US_03.03 Bookings management link', () => {
         cy.fixture('leftMenuPanel/menuLinks').then(link => {
             this.link = link;
         });
+        cy.fixture('bookingsListPage/url').then(url => {
+            this.url = url;
+        });
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:19]: Delete.

File: cypress/fixtures/bookingsListPage/url.json [hunk 10]
```diff
@@ -0,0 +1,3 @@
+{
+    "bookingsManagementUrl": "https://qatest.site/orders/"
+}
```
Comment (Rom-T) [inline cypress/fixtures/bookingsListPage/url.json:3]: Delete file url.json

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js (hunk 7 above)
Comment (AlexMokeev) [reply cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:1]: ok

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js (hunk 9 above)
Comment (AlexMokeev) [reply cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:19]: ok

File: cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js (hunk 8 above)
Comment (AlexMokeev) [reply cypress/e2e/testUiAndFunction/leftMenuPanel/menuPanelSection/us_03.03_LeftMenuPanel_BookingsManagementLink.cy.js:39]: ok

File: cypress/fixtures/bookingsListPage/url.json (hunk 10 above)
Comment (AlexMokeev) [reply cypress/fixtures/bookingsListPage/url.json]: Ok

**Pull request merged**